controllers/postController.js

```diff
@@ -21,9 +21,11 @@ const createPost = async (req,res) =>{
     //     await cloudinary.uploader.upload(file.path,{folder:"media_url"}).secure_url
     // })
 
-    const filePath = req.file.path
-    const cloudinaryPath = await cloudinary.uploader.upload(filePath,{folder:"media_url"})
-    body.media_url = cloudinaryPath.secure_url
+    const filePath = req.file?.path
```

Comment:
line 24. I'm guessing the '?' is a typo. Right?

Reply:
No it’s not , it’s called optional chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
you can read about it here

Reply:
oh okay, thanks